pygfx/renderers/wgpu/meshshader.py

```diff
@@ -234,11 +280,19 @@ def code_vertex(self):
 
             // For the wireframe we also need the ndc_pos of the other vertices of this face
             $$ if wireframe
-                $$ for i in (1, 2, 3)
-                    let raw_pos{{ i }} = load_s_positions(i32(ii[{{ i - 1 }}]));
-                    let world_pos{{ i }} = world_transform * vec4<f32>(raw_pos{{ i }}, 1.0);
-                    let ndc_pos{{ i }} = u_stdinfo.projection_transform * u_stdinfo.cam_transform * world_pos{{ i }};
-                $$ endfor
+                $$ if indexer == 3
+                    $$ for i in (1, 2, 3)
+                        let raw_pos{{ i }} = load_s_positions(i32(ii[{{ i - 1 }}]));
+                        let world_pos{{ i }} = world_transform * vec4<f32>(raw_pos{{ i }}, 1.0);
+                        let ndc_pos{{ i }} = u_stdinfo.projection_transform * u_stdinfo.cam_transform * world_pos{{ i }};
+                    $$ endfor
+                $$ elif indexer == 6
+                    $$ for i in (1, 2, 3, 4)
+                        let raw_pos{{ i }} = load_s_positions(i32(ii[{{ i - 1 }}]));
+                        let world_pos{{ i }} = world_transform * vec4<f32>(raw_pos{{ i }}, 1.0);
+                        let ndc_pos{{ i }} = u_stdinfo.projection_transform * u_stdinfo.cam_transform * world_pos{{ i }};
+                    $$ endfor
+                $$ endif
```

Comment:
Would something like this work? Otherwise we can introduce a template var `verts_per_face`.
```suggestion
            $$ for i in ((1, 2, 3) if indexer == 3 else (1, 2, 3, 4))
                let raw_pos{{ i }} = load_s_positions(i32(ii[{{ i - 1 }}]));
                let world_pos{{ i }} = world_transform * vec4<f32>(raw_pos{{ i }}, 1.0);
                let ndc_pos{{ i }} = u_stdinfo.projection_transform * u_stdinfo.cam_transform * world_pos{{ i }};
            $$ endfor
```